app/controllers/users_controller.rb

```diff
@@ -0,0 +1,65 @@
+class UsersController < ApplicationController
+  before_action :authorized, only: [:show]
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)
+    if @user.valid?
+      @user.save
+      redirect_to @user
+    else
+      redirect_to root_path
+    end
+  end
+
+  def show
+    user
```

Comment:
This line is unnecessary now. Whenever you call `@user` is what should be getting replaced with this new method.